src/ZfSimpleMigrations/Library/Migration.php

```diff
@@ -288,7 +288,12 @@ protected function applyMigration(array $migration, $down = false, $fake = false
                 $sqlList = $down ? $migrationObject->getDownSql() : $migrationObject->getUpSql();
                 foreach ($sqlList as $sql) {
                     $this->outputWriter->writeLine("Execute query:\n\n" . $sql);
-                    $this->connection->execute($sql);
+                    /** @var \Zend\Db\Adapter\Driver\Pdo\Statement $statement */
+                    $statement = $this->connection->prepare($sql);
+                    $statement->execute();
+                    /** @var \PDOStatement $statement */
+                    $statement = $statement->getResource();
+                    while ($statement->nextRowset()) {/* https://bugs.php.net/bug.php?id=61613 */}
```

Comment:
Multiple drivers do not support multiple rowsets. As such this change makes this module unusable for multiple drivers (including sqlite). Should either revert back or add conditional logic to execute nextRowset only if the current driver supports the call.